datamade/just-spaces
# Strip out the frontend app

## Overview

Closes #187.

Remove the frontend app introduced in #59 since we won't be using it for this iteration of the project.

## Testing Instructions

 * Confirm tests pass on Travis
 * Spin up the dev server locally and confirm that it loads without error
 * Double-check the instructions in the README to make sure they match the dev workflow
